libr/anal/esil.c

```diff
@@ -105,17 +105,28 @@ R_API RAnalEsil *r_anal_esil_new(int stacksize, int iotrap, unsigned int addrsiz
 	return esil;
 }
 
-R_API int r_anal_esil_set_op(RAnalEsil *esil, const char *op, RAnalEsilOp code) {
+R_API bool r_anal_esil_set_op(RAnalEsil *esil, const char *op, RAnalEsilOpCb code, ut32 push, ut32 pop, ut32 type) {
+	r_return_val_if_fail(code && op && strlen (op) && esil && esil->ops, false);
```

Comment:
Missing space

Reply:
May be the reason why travis is failing. Run the tests in local and see if now is failing bwcause of that assert